docs/getting-started/overview.rst

```diff
@@ -1,6 +1,45 @@
-.. include:: /stub-topic.txt
+Overview of different editions of .NET
+======================================
+
+With the advent of .NET Core and .NET Native, the layout of the .NET ecosystem has changed significantly. Where before there was a (somewhat) simple mapping between platforms and .NET Framework versions, today we have a lot of 
+
+.NET Framework
+--------------
+The tried and true workhorse of the past 10+ years, the full .NET framework is still around, of course, and has been recently updated to version 4.6. There are many enhancements to the CLR (the runtime) and the BCL (the class library) that comes with this release for the consumers presented in the diagram above, such as the .NET Compiler Platform, new language innovations etc.
+
+This edition of the .NET Framework is not cross-platform and you can run it only on Windows. Most of the consumers of this particular edition of the .NET Framework should be already know, if in name only. Windows Presentation Foundation or WPF is a framework for building desktop applications using specific technologies to make them compelling and rich. Windows Forms is already known as the bulwark of the LOB desktop applications from the early day of the .NET Framework. Of course, ASP.NET is Microsoft's main web stack, which is getting a major overhaul in the 5 release timeframe.
+
+.NET Core
+---------
+.NET Core is a cloud-optimized, pay-as-you-go, cross-platform port of the .NET Framework. The roadmap currently in place covers support for three main operating systems, Linux, Windows and OS X. The community porters have made it work on FreeBSD as well. It is freely available on `GitHub <https://www.github.com/dotnet/coreclr>`_ and it is very easy to get for the supported systems and get going.
+
+Today, development of modern services and applications is very different from what was available a couple of years ago. Cloud services, application containers with Docker, strong client-side requirements and much more all change the way in which a modern developer may get about doing their everyday tasks. Many of these new developments mean that the developer wants to have a fully packaged environment that can be deployed in isolation, without affecting anything else, not to mention a way to include only those things that the application in question actually needs. 
```

Comment:
SPAs, modern JS frameworks, those kinds of things that weren't really in vogue a couple of years back (even if they existed). Also, client-side package managers and task runners (bower, grunt, gulp).